diff --git a/dist/extract-experimental/buildExternalizeFilter.js b/dist/extract-experimental/buildExternalizeFilter.js
index f836f0f9840a4ec6f382695bb42706fb9ef4292a..744afbce700bb56fb4429158a35ac4c511e0cc83 100644
--- a/dist/extract-experimental/buildExternalizeFilter.js
+++ b/dist/extract-experimental/buildExternalizeFilter.js
@@ -1,6 +1,9 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.getPackageJson = exports.buildExternalizeFilter = void 0;
+
+const fs = require('node:fs')
+
 function createPackageRegExp(packageName) {
     return new RegExp("^" + packageName + "(?:\\/.+)?");
 }
@@ -33,6 +36,14 @@ async function getPackageJson(rootDir) {
         throw new Error("We could not able to find your package.json file. " +
             "Check that `rootDir` is pointing to the folder with package.json");
     }
-    return await import(packageJsonPath);
+    try {
+        return JSON.parse(await fs.promises.readFile(packageJsonPath, "utf-8"))
+    } catch (e) {
+        throw new Error(
+            `Unable to read package.json file at path ${packageJsonPath}. \n\n Error: ${
+                e.message
+            }`
+        )
+    }
 }
 exports.getPackageJson = getPackageJson;
